sdk/validatormanager/validator_manager_poa.go

```diff
@@ -23,25 +23,46 @@ func PoAValidatorManagerInitialize(
 	privateKey string,
 	subnetID ids.ID,
 	ownerAddress common.Address,
+	useACP99 bool,
```

Comment:
We don't have to differentiate PoAValidatorManagerInitialize and PoSValidatorManagerInitialize anymore. We can just have one function InitializeValidatorManager and have it in root.go / some common file